src/include/sof/audio/component.h

```diff
@@ -246,7 +248,7 @@ struct comp_dev {
 	uint16_t state;		   /**< COMP_STATE_ */
 	uint64_t position;	   /**< component rendering position */
 	uint32_t frames;	   /**< number of frames we copy to sink */
-	uint32_t output_rate;      /**< 0 means all output rates are fine */
+	uint32_t output_rate;	   /**< 0 means all output rates are fine */
```

Comment:
more unrelated drive-by changes above

Reply:
You mean that all those cosding style changes should be in separate commit?

Reply:
@bkokoszx Yup, they shouldn't be mixed with actual functional changes.